src/main/java/org/broadinstitute/dsm/export/WorkflowAndFamilyIdExporter.java

```diff
@@ -0,0 +1,137 @@
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
+
+public class WorkflowAndFamilyIdExporter implements Exporter {
+
+    private static final Logger logger = LoggerFactory.getLogger(WorkflowAndFamilyIdExporter.class);
+    private static final Gson gson = new Gson();
+    private static final ParticipantDataDao participantDataDao = new ParticipantDataDao();
+    public static final String RGP_PARTICIPANTS = "RGP_PARTICIPANTS";
+
+    @Override
+    public void export(int instanceId) {
+
+    }
+
+    public void export(int instanceId, AtomicBoolean clearBeforeUpdate) {
+        logger.info("Started exporting workflows and family ID-s for instance with id " + instanceId);
+        List<String> workFlowColumnNames = findWorkFlowColumnNames(instanceId);
+        List<ParticipantDataDto> allParticipantData = participantDataDao.getParticipantDataByInstanceid(instanceId);
+        checkWorkflowNamesAndExport(workFlowColumnNames, allParticipantData, instanceId, clearBeforeUpdate);
+        logger.info("Finished exporting workflows and family ID-s for instance with id " + instanceId);
+    }
+
+    private List<String> findWorkFlowColumnNames(int instanceId) {
+        FieldSettingsDao fieldSettingsDao = FieldSettingsDao.of();
+        List<FieldSettingsDto> fieldSettings = fieldSettingsDao.getFieldSettingsByInstanceId(instanceId);
+        List<String> workflowColumns = new ArrayList<>();
+        for (FieldSettingsDto fieldSetting: fieldSettings) {
+            String actions = fieldSetting.getActions();
+            if (actions != null) {
+                Value[] actionsArray =  gson.fromJson(actions, Value[].class);
+                for (Value action : actionsArray) {
+                    if (ESObjectConstants.ELASTIC_EXPORT_WORKFLOWS.equals(action.getType())) {
+                        workflowColumns.add(fieldSetting.getColumnName());
+                        break;
+                    }
+                }
+            }
+        }
+        return workflowColumns;
+    }
+
+    public void checkWorkflowNamesAndExport(List<String> workFlowColumnNames, List<ParticipantDataDto> allParticipantData,
+                                                   int instanceId, AtomicBoolean clearBeforeUpdate) {
+        DDPInstance ddpInstance = DDPInstance.getDDPInstanceById(instanceId);
+        Optional<String> maybeEsParticipantIndex =
+                new DDPInstanceDao().getEsParticipantIndexByInstanceId(instanceId);
+        for (ParticipantDataDto participantData: allParticipantData) {
+            String data = participantData.getData();
+            if (data == null) {
+                continue;
+            }
+            String ddpParticipantId = participantData.getDdpParticipantId();
+            if (!ddpParticipantId.equals("WUKIOQNKXJZGCAXCSYGB")) {
```

Comment:
What ddp_participant_id is that ???

Reply:
I deleted that line. Used for testing, forgot to delete before